discord/ext/commands/bot.py

```diff
@@ -94,7 +96,9 @@ def __repr__(self):
 
 _default = _DefaultRepr()
 
-class BotBase(GroupMixin):
+_CT = TypeVar('_CT', bound=Context)
```

Comment:
was this supposed to go in discord/ext/commands/bot.py? I see the same code in the pyi file.